code/modules/mob/living/simple_animal/hostile/mining/abandoned_minebot.dm

```diff
@@ -0,0 +1,88 @@
+/mob/living/simple_animal/hostile/asteroid/abandoned_minebot
+	name = "abandoned minebot"
+	desc = "The instructions printed on the side are faded, and the only thing that remains is mechanical bloodlust."
+	icon = 'icons/obj/aibots.dmi'
+	icon_state = "mining_drone_offense"
+	icon_living = "mining_drone_offense"
+	icon_aggro = "mining_drone_alert"
+	mob_biotypes = MOB_ROBOTIC
+	vision_range = 6
+	aggro_vision_range = 7
+	status_flags = CANPUSH
+	mouse_opacity = MOUSE_OPACITY_ICON
+	weather_immunities = list("ash")
+	move_to_delay = 10
+	maxHealth = 75
+	health = 75
+	melee_damage_lower = 15
+	melee_damage_upper = 15
+	obj_damage = 10
+	environment_smash = ENVIRONMENT_SMASH_STRUCTURES
+	check_friendly_fire = TRUE
+	stop_automated_movement_when_pulled = TRUE
+	attacktext = "drills into"
+	attack_sound = 'sound/weapons/circsawhit.ogg'
+	projectiletype = /obj/item/projectile/kinetic
+	projectilesound = 'sound/weapons/kenetic_accel.ogg'
+	ranged_cooldown_time = 30
+	ranged_message = "fires"
+	speak_emote = list("states")
+	del_on_death = TRUE
+	light_range = 5
+	var/sight_flags = SEE_TURFS
```

Comment:
Sight flags isnt used anywhere here. I recommend you check the other vars you got from the minebot too
```suggestion
```